setup.cfg

```diff
@@ -1,25 +0,0 @@
-[bdist_rpm]
-requires=h5py pyresample python2-numexpr pyhdf xarray dask h5netcdf
-release=1
-doc_files = doc/Makefile doc/source/*.rst doc/examples/*.py
-
-[bdist_wheel]
-universal=1
-
-[flake8]
-max-line-length = 120
-exclude =
-    satpy/readers/li_l2.py
-    satpy/readers/scatsat1_l2b.py
-    satpy/version.py
-    satpy/tests/features
-per-file-ignores =
-    satpy/tests/*/conftest.py:F401
-    satpy/tests/*/*/conftest.py:F401
-    doc/source/doi_role.py:D103
-    satpy/tests/features/steps/*.py:F811
-
-[coverage:run]
-relative_files = True
-omit =
-    satpy/version.py
```

Comment:
Darn it @mraspaud. This is why coveralls is broken. The coverage config never got transferred to pyproject.toml. Fixing now...

Reply:
Sorry